Test/ComponentTestWith_bUnit.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using Xunit;
+using Bunit;
+using CDNApplication.TCComponents;
+using CDNApplication.TestHelper;
+using CDNApplication.Pages;
+using System.Linq;
+using AngleSharp.Diffing.Extensions;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Components;
+using System.Runtime.InteropServices.WindowsRuntime;
+
+namespace Test
+{
+    public class ComponentTestWith_bUnit
```

Comment:
```suggestion
    public class TCInputSelectTest 
```
I would rename to something like this since that is the class we are testing.